command.go

```diff
@@ -460,6 +462,7 @@ func parseShellArgs(args []string) (Command, error) {
 	flag.String("assume", "", "Role to assume")
 	flag.Bool("no-session", false, "Disable use of temporary credentials")
 	flag.Bool("refresh", false, "Start a new session with new temporary credentials and a refreshed expiration")
+	flag.String("region", os.Getenv("AWS_DEFAULT_REGION"), "The AWS region to use to generate STS credentials")
```

Comment:
Same comment here about `AWS_REGION` vs `AWS_DEFAULT_REGION`

Reply:
Adopting as noted in above comment.